03-task-manager/starter/routes/tasks.js

```diff
@@ -6,9 +6,15 @@ const {
   getTask,
   updateTask,
   deleteTask,
+  // replaaceTask,
 } = require(`../controllers/tasks`);
 
 router.route("/").get(getAllTasks).post(createTask);
-router.route("/:id").get(getTask).patch(updateTask).delete(deleteTask);
+router
+  .route("/:id")
+  .get(getTask)
+  .patch(updateTask)
+  .delete(deleteTask)
+  // .put(replaceTask);
```

Comment:
fun, were you testing PUT methods?

Reply:
Yes! I was following along with the instructor and wanted to leave the PUT and some of the other sandbox code for future reference. Is that a poor practice for an entry level coder?